lexicons/tools/ozone/setting/listOptions.json

```diff
@@ -0,0 +1,60 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.setting.listOptions",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "List settings with optional filtering",
+      "parameters": {
+        "type": "params",
+        "properties": {
+          "limit": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 100,
+            "default": 50
+          },
+          "cursor": {
+            "type": "string"
+          },
+          "scope": {
+            "type": "string",
+            "knownValues": ["instance", "personal"],
+            "default": "instance"
+          },
+          "prefix": {
+            "type": "string",
+            "description": "Filter keys by prefix"
+          },
+          "keys": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "nsid"
+            },
+            "description": "Filter for only the specified keys. Ignored if prefix is provided"
+          }
```

Comment:
Often nice to have a max length on array params, even if just to help keep the caller under url length limits.

Reply:
yep yep. i had this elsewhere but missed a few spot ig